wagtail/wagtailcore/models.py

```diff
@@ -207,6 +207,14 @@ def get_page_models():
     return PAGE_MODEL_CLASSES
 
 
+def get_default_page_content_type():
+    """
+    Returns the content type to use as a default for pages which their
```

Comment:
I don't understand this sentence - should it be 'when their content type'? or 'whose content type'?